adam-cli/src/main/scala/org/bdgenomics/adam/cli/ADAMMain.scala

```diff
@@ -69,6 +70,17 @@ object ADAMMain extends Logging {
       )
     )
 
+  def main(args: Array[String]) {
+    new ADAMMain(defaultCommandGroups).apply(args)
```

Comment:
If the `@Inject` command supports `Option`, it would be nicer to have `class ADAMMain @Inject() (injectedGroups: Option[List[CommandGroup]]) {` or somesuch. I'll stop now.